File: app/src/main/java/se/insektionen/songbook/SongbookApplication.java
package se.insektionen.songbook;

import android.app.Application;

import java.util.Locale;

import se.insektionen.songbook.services.SharedHttpClient;

/**
 * Main application class.
 */
public final class SongbookApplication extends Application {
	@Override
	public void onCreate() {
		super.onCreate();
		Locale.setDefault(new Locale("sv", "SE"));
		SharedHttpClient.initializeCache(getApplicationContext());
	}
}

File: app/src/main/java/se/insektionen/songbook/SongbookApplication.kt
package se.insektionen.songbook

import android.app.Application
import se.insektionen.songbook.services.SharedHttpClient
import java.util.Locale

/**
 * Main application class.
 */
class SongbookApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        Locale.setDefault(Locale("sv", "SE"))
        SharedHttpClient.initialize(applicationContext)
    }
}
